### DISCUSSION TITLE
Slider autorotation is must?!

### DISCUSSION THREAD

**Discussion by Sa7aR-Kassem** [TDRA-ae/aegov-dls#79] [General]
Is there any restriction that the slider must be set to auto-rotate, meaning that it should start over again once it finishes the current slides? Because from what I’ve seen, all the demos in the  [Design System](https://designsystem.gov.ae/docs/components/slider) are set up this way.

**Comment by moiz-conceptlz** [comment]
Hello @Sa7aR-Kassem 

There is no such restriction to auto-rotate. It is just good UX to have content that was first to come back in focus when the elements in the slider are longer and to not have a user go back to the first element manually.
